lib/views/help/glossary.html.erb

```diff
@@ -138,6 +138,13 @@ Environmental Information Regulations 2004 (
 <dl>
 <dt id="gdpr">GDPR</dt>
 <dd>General Data Protection Regulation - a law about data protection and privacy. Following Brexit, the UK has its own version of GDPR. See also <a href="https://www.legislation.gov.uk/eur/2016/679/contents">the text of the General Data Protection Regulation</a>.</dd>
+</dl>
+ <dl>
+<dt id="github">GitHub</dt>
+<dd>GitHub is an online tool for software development and version control. 
+ Issues can be raised in GitHub where there are bugs in the site software or to suggest improvements or new features. 
+  The code behind the WhatDoTheyKnow website is worked on using GitHub. See also <a fre="#alaveteli>Alaveteli</a>.
```

Comment:
typo: `<a fre=`

Reply:
This is from a previous commit in a different PR which has already been merged so I have fixed in faf1be82990d86f176d20e2488507f75aebf2f08

Nice spot